### ISSUE TITLE
Replace `Wavefront` padding implementation with `jax.numpy.pad`

### ISSUE THREAD

**Issue by LouisDesdoigts** [LouisDesdoigts/dLux#130]
In the interest of making the code simpler/more readable this would likely be a good idea.

https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.pad.html

**Comment by LouisDesdoigts** [comment]
Completed with #246
